etc/kayobe/kolla/config/cinder/ceph.conf

```diff
@@ -0,0 +1,7 @@
+[global]
+fsid = 822209f7-5dfb-427c-b3d0-582e8549a218
+mon host = 10.12.5.5,10.12.5.6,10.12.5.7
+mon initial members = ceph-0,ceph-1,ceph-2
+auth_cluster_required = cephx
+auth_service_required = cephx
+auth_client_required = cephx
```

Comment:
At GR we found that some of the nova config was also required, as it gets used when mounting the volume.

Reply:
I think we kept `nova_backend_ceph = no`, and did not use ceph.client.nova.keyring but had ceph.client.cinder.keyring in etc/kayobe/kolla/config/nova

Reply:
There may have been changes in stackhpc/queens for kolla-ansible also to make this work, not sure if they made it to stackhpc/pike.

Reply:
It's 355ce7e539feb02ef51d8e6ea5b89fc7541948d6 on kolla-ansible stackhpc/queens, looks like it needs a port to stackhpc/pike.